packages/glimmer-runtime/lib/builder.ts

```diff
@@ -89,18 +91,31 @@ class GroupedElementOperations implements ElementOperations {
   }
 
   addAttribute(name: InternedString, reference: PathReference<string>) {
-    if (this.element.tagName === 'INPUT' && name === 'value') {
-      this.group.push(new EmptyStringResetingProperty(this.element, name, reference));
+    let { element } = this;
+    let { tagName } = element;
+
+    if (SanitizedNonNamespacedAttribute.requiresSanitization(tagName, name)) {
+      this.group.push(new SanitizedNonNamespacedAttribute(element, name, reference));
+    } else if (tagName === 'INPUT' && name === 'value') {
+      this.group.push(new EmptyStringResetingProperty(element, name, reference));
+    } else {
+      this.group.push(new NonNamespacedAttribute(element, name, reference));
     }
```

Comment:
I changed the conditions here. This used to push two objects in the case of `input[value]`. Was this the correct behavior? For the sanitized case it should be one or the other, not both.